cc_pseudo_crawl/extract_text/extract_text_and_html_metadata.py

```diff
@@ -0,0 +1,447 @@
+import copy
+import logging
+import os
+import re
+import subprocess
+import sys
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Tuple
+from urllib.parse import unquote, urlparse, urlsplit
+
+import datasets
+import hydra
+import wandb
+from datasets import Dataset, Features, Value, config, load_dataset, load_from_disk
+from hydra.core.config_store import ConfigStore
+from omegaconf import OmegaConf
+
+from bsmetadata.preprocessing_tools import html_parser
+from bsmetadata.train import show_help
```

Comment:
You should add this in `cc_pseudo_crawl/requirements.txt`

Reply:
It's in `cc_pseudo_crawl/extract_text/requirements.txt`